packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -26,6 +26,33 @@ export const toggleGroupControl = ( {
 
 	${ toggleGroupControlSize( size ) }
 	${ ! isDeselectable && enclosingBorders( isBlock ) }
+
+
+	@media not ( prefers-reduced-motion ) {
+		&.is-animation-enabled::before {
+			transition-property: left, width;
+			transition-duration: 0.2s;
+			transition-timing-function: ease-out;
+		}
+	}
+
+	&::before {
+		content: '';
+		position: absolute;
+		pointer-events: none;
+		background: ${ COLORS.gray[ 900 ] };
```

Comment:
Let's take this opportunity to use `COLORS.theme.gray` instead

Reply:
Done in fe2b51e9f17be939d789c4f7dc14ec6771876702